From 6ec760256c3f6789a7c918b91bd2fe2c78a60849 Mon Sep 17 00:00:00 2001
From: Luciano Chavez <chavez@us.ibm.com>
Date: Tue, 22 Jul 2025 15:37:39 -0500
Subject: [PATCH] scripts/ppc64_diag_mkrsrc: Fix typo to avoid syntax error

A user executed the diag_nvme utility to inject an error in a disk in
order to generate a servicelog event. While executing the command
and after the event was logged, a syntax error was displayed for
the ppc64_diag_mkrsrc script for the line with:

"`cat /proc/device-tree/system-id | cut -c5- 2>/dev/null` :"

which showed the statement terminated with a `:` rather than a `;`.
The script was then aborted and did not generate a IBM.ServiceEvent
RMC resource as expected.

Signed-off-by: Luciano Chavez <chavez@us.ibm.com>
Reviewed-by: Sathvika Vasireddy <sv@linux.ibm.com>
---
 scripts/ppc64_diag_mkrsrc | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/scripts/ppc64_diag_mkrsrc b/scripts/ppc64_diag_mkrsrc
index c59dd8e..ce7d013 100755
--- a/scripts/ppc64_diag_mkrsrc
+++ b/scripts/ppc64_diag_mkrsrc
@@ -132,7 +132,7 @@ chop $cec_machinetype;
 
 $cec_machineserial = -e '/proc/device-tree/ibm,vendor-system-id' ?
 `cat /proc/device-tree/ibm,vendor-system-id | cut -c5- 2>/dev/null` :
-`cat /proc/device-tree/system-id | cut -c5- 2>/dev/null` :
+`cat /proc/device-tree/system-id | cut -c5- 2>/dev/null` ;
 
 chomp $cec_machineserial;
 chop $cec_machineserial;
-- 
2.43.0

